pyuvdata/uvcal.py

```diff
@@ -221,8 +221,6 @@ def __init__(self):
                 'if cal_type is delay, shape is (Nspws, 1, Ntimes, Njones), '
                 'type = float.')
         self._total_quality_array = uvp.UVParameter('total_quality_array', description=desc,
-                                                    form=('Nspws', 'Nfreqs',
```

Comment:
Okay, I've added the default shape back in. I've also added a test to make sure the array is the right size.